src/routes/transactions/converters/mod.rs

```diff
@@ -21,7 +21,7 @@ use crate::utils::{SAFE_TRANSFER_FROM_METHOD, TRANSFER_FROM_METHOD, TRANSFER_MET
 use rocket::futures::future::OptionFuture;
 
 impl SafeTransaction {
-    async fn transaction_info(
+    pub(crate) async fn transaction_info(
```

Comment:
This is now used by the preview endpoint so I had to expose this to the other crate. It might make sense to review the project structure so that we know what can/should be public to the other crates